packages/app/src/app/pages/Pro/components/Switcher.tsx

```diff
@@ -56,12 +61,25 @@ export const Switcher: React.FC<{
             name={activeTeamInfo.name}
           />
 
-          <Stack css={{ marginLeft: 24}} direction="vertical">
-            <WorkspaceName>
-              <span>{activeTeamInfo.name}</span>
-            </WorkspaceName>
+          <Stack css={{ marginLeft: 24 }} direction="vertical">
+            <Stack align="center" gap={1}>
+              <WorkspaceName>
+                <span>{activeTeamInfo.name}</span>
+              </WorkspaceName>
+              {isFreeWorkspace && (
+                <Badge
+                  color="accent"
+                  css={{
+                    height: 'fit-content',
```

Comment:
Can't check it right now, but `fit-content` isn't supported for Firefox according to MDN, will this cause issues on FF? I'll come back to this if I'm able to test again!

Reply:
It seems to work, but removing the height style seems to look the same?

Reply:
I would also avoid passing generic props/css to certain components which should be "closed" like the badge, to ensure we don't abuse this further down the line